maintenance_plan/models/maintenance_equipment.py

```diff
@@ -38,9 +38,13 @@ def _prepare_request_from_plan(self, maintenance_plan,
         team = maintenance_plan.maintenance_team_id or self.maintenance_team_id
         if not team:
             team = self.env['maintenance.request']._get_default_team_id()
+
+        description = self.name if self else maintenance_plan.name
+        name = _('Preventive Maintenance (%s) - %s') % (
+            maintenance_plan.maintenance_kind_id.name, description)
```

Comment:
maintenance_kind_id is required?

Reply:
No its not, I just fixed this part so it doesn't display `False` on the title.